shards/core/runtime.cpp

```diff
@@ -1025,17 +1034,18 @@ void validateConnection(InternalCompositionContext &ctx) {
     }
 
     if (!matching) {
-      std::string error;
```

Comment:
Can you change back to the shorter format (the fmt one)

Reply:
It basically just doesn't log all the exposed variables and makes the error message a bit more concise